src/creatures/monsters/monsters.cpp

```diff
@@ -401,6 +401,15 @@ bool Monsters::deserializeSpell(const pugi::xml_node& node, spellBlock_t& sb, co
 
 			Condition* condition = Condition::createCondition(CONDITIONID_COMBAT, CONDITION_DRUNK, duration, 0);
 			combat->addCondition(condition);
+		} else if (tmpName == "fear") {
```

Comment:
This not is more used, is from old XML monsters loading.

Reply:
> Should I remove this bit or the whole function?

You can remove everything if you want.

Reply:
I think I will leave it as is, I tried to remove only this method but seems that there are more parts of the code that may need to change.

I believe this may be its own PR to remove this function and other related ones.